packages/palette/src/elements/Menu/Menu.tsx

```diff
@@ -43,9 +43,7 @@ export const Menu: React.FC<MenuProps> = ({
             </Box>
           )}
 
-          <Flex flexDirection="column" pt={title ? 0 : padding}>
-            {children}
-          </Flex>
+          <Flex flexDirection="column">{children}</Flex>
```

Comment:
Okay this attempts to simplify conditionally adjusting the padding. Previously we were giving the box container for the title 20px of top padding. Then conditionally adding 0px of top padding to the menu content flex container when there was a title present and 10px when there wasn't. This refactor allows `BorderBox`  to handles the top and bottom padding. 

Does this make sense @sweir27 ?

Reply:
Makes a lot more sense! Nice refactor.